src/Bundler.js

```diff
@@ -713,7 +713,11 @@ class Bundler extends EventEmitter {
 
   async serve(port = 1234, https = false) {
     this.server = await Server.serve(this, port, https);
-    this.bundle();
```

Comment:
Previously if this line was throwing an error we had a silent unhandled promise rejection